src/Catty UITests/Defines/LanguageTranslationDefinesUI.swift

```diff
@@ -586,13 +590,13 @@ let kUIFEFunctionJoin = NSLocalizedString("join", bundle: Bundle(for: LanguageTr
 let kUIFEFunctionLength = NSLocalizedString("length", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionFloor = NSLocalizedString("floor", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionCeil = NSLocalizedString("ceil", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionNumberOfItems = NSLocalizedString("number_of_items", bundle: Bundle(for: LanguageTranslation.self), comment: "")
+let kUIFEFunctionNumberOfItems = NSLocalizedString("number of items", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionElement = NSLocalizedString("element", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 let kUIFEFunctionContains = NSLocalizedString("contains", bundle: Bundle(for: LanguageTranslation.self), comment: "")
 
-let kUIFEFunctionScreenIsTouched = NSLocalizedString("screen_is_touched", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionScreenTouchX = NSLocalizedString("screen_touch_x", bundle: Bundle(for: LanguageTranslation.self), comment: "")
-let kUIFEFunctionScreenTouchY = NSLocalizedString("screen_touch_y", bundle: Bundle(for: LanguageTranslation.self), comment: "")
+let kUIFEFunctionScreenIsTouched = NSLocalizedString("stage is tapped", bundle: Bundle(for: LanguageTranslation.self), comment: "")
```

Comment:
I think this is called "touched" in Catroid (https://github.com/Catrobat/Catroid/pull/3018/files#diff-83cbece94e949621177890c9a504eec6R1747). Could you please check that. Thank a lot!